assets/theme-css/pst/content/_code.scss

```diff
@@ -12,6 +12,24 @@ div.literal-block-wrapper {
   flex-direction: column;
   width: unset;
   border-radius: $admonition-border-radius;
+
+  // Copy-to-clipboard button.
+  & .copy-button {
```

Comment:
Note: we cannot modify the pst theme, so this will have to go elsewhere. Better if we can re-use what they have, whenever possible.

Reply:
Ah, sorry, I keep accidentally editing the wrong files this way.   :)  Will fix.  Thanks.